spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -247,19 +260,61 @@ def _measure_within_im(self, im_lesion, im_ref, label_lst):
                 type='info')
 
     def _measure_volume(self, im_data, p_lst, idx):
+        """
+        Measure the volume of the lesion
+        """
         for zz in range(im_data.shape[2]):
             self.volumes[zz, idx - 1] = np.sum(im_data[:, :, zz]) * p_lst[0] * p_lst[1] * p_lst[2]
 
         vol_tot_cur = np.sum(self.volumes[:, idx - 1])
         self.measure_pd.loc[idx, 'volume [mm3]'] = vol_tot_cur
         printv('  Volume : ' + str(np.round(vol_tot_cur, 2)) + ' mm^3', self.verbose, type='info')
 
+    def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
+        """
+        Measure the maximum axial damage ratio
+        The axial damage ratio is calculated as the ratio of lesion area divided by spinal cord area
+        The axial damage ratio is calculated for each slice and then the maximum value is retained
+        REF: Smith, A. C. et al. (2021). Axial MRI biomarkers of spinal cord damage to predict future walking and motor
+        function: A retrospective study. Spinal Cord, 59(6), 693-699. https://doi.org/10.1038/s41393-020-00561-w
+        """
+
+        # Load the spinal cord image
+        im_sc = Image(self.fname_sc)
+        im_sc_data = im_sc.data
+
+        # Get the lesion and spinal cord area
+        axial_damage_ratio_dict = {}
+        # Get slices with lesion
+        lesion_slices = np.unique(np.where(im_data)[2])
+        for slice in lesion_slices:
+            # Lesion area
+            lesion_area = np.sum(im_data[:, :, slice]) * p_lst[0] * p_lst[1]
+            # Spinal cord area
+            sc_area = np.sum(im_sc_data[:, :, slice]) * p_lst[0] * p_lst[1]
+            # Compute the axial damage ratio slice by slice
+            axial_damage_ratio_dict[slice] = lesion_area / sc_area
```

Comment:
I notice that the area computation is based on z slices, and so it doesn't take into account the angle correction like the other functions (length/diameter) do. 

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/da9ac4023d4dec1d9bd1d940cd2eba0785eff15a/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L320-L325

Is this something we want to consider for consistency sake?

Reply:
> I notice that the area computation is based on z slices, and so it doesn't take into account the angle correction like the other functions (length/diameter) do.

This is an excellent point! Thank you for pointing this out. Indeed, I was considering the angle correction. Eventually, I decided not to use it because the `axial damage ratio` is just a ratio of lesion and spinal cord masks.  So I assumed that since both lesion and spinal cord masks would be corrected using the same angle correction, it would not impact the resulting `axial damage ratio`.